app/common/state/tabContentState/partitionState.js

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// State
+const frameStateUtil = require('../../../../js/state/frameStateUtil')
+
+// Constants
+const {tabs} = require('../../../../js/constants/config')
+
+module.exports.isPartitionTab = (state, frameKey) => {
+  const frame = frameStateUtil.getFrameByKey(state, frameKey)
+
+  if (frame == null) {
+    return false
+  }
+
+  return frame.get('partitionNumber')
+}
+
+module.exports.getPartitionNumber = (state, frameKey) => {
```

Comment:
they're slightly different. I'll keep that one to avoid passing `frame.get('partitionNumber')` as a param. There's also other functions depending on this and all tested. I'll buy the next pancake round if you let this as nit :P